source/scripts/add.js

```diff
@@ -106,12 +106,15 @@ function addTask(data) {
   // tableRow.append(tableRowNotes);
   document.body.querySelector("tbody").append(tableRowNotes);
 
-  // display notes when clicked, hide when clicked again
+  // Display notes when clicked, hide when clicked again
```

Comment:
Great job on showing the notes when clicking each row! It's really cool. I noticed that notes will also show up when click the edit button. Is that what we want? It confused me for a sec since I'm not expecting the notes show up when I want to edit the task.

Reply:
It's probably because you didn't click on the edit button but somewhere within the action cell. We could change it so that edit function doesn't pertain to just the icon button but the entire data cell (`td`). I think this doesn't have to be fixed in this pull request especially if we want to update our live website with the function before the video tomorrow, but depending on what team leads or @melvyn9 @nrejai think, we could make a quick fix or create a new issue for this since we never explicitly clarified.

Reply:
Good catch Tianya! Yes it is not supposed to do that. When we implemented the task drop down, we made it so that as long as anywhere in the task row is clicked, then it will drop down the notes. However, we forgot that since the edit button is on the same row, clicking it will cause the notes to drop down as well as the edit form to pop up. I have created an issue for this bug. It is issue #72